### PR TITLE
Update Layout System, renew keypad, update ViewController present method

### PR DESCRIPTION
**Pull request by dannylin0711**
Update Layout System, renew keypad, update ViewController present method, Card page not yet update to the new present method

### CODE REVIEW

File: SpiceCompanion/Spice.entitlements [hunk 1]
```diff
@@ -2,8 +2,13 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>applinks:example.com</string>
```
Comment (rodepanda) [inline SpiceCompanion/Spice.entitlements:7]: This is unused by the new code and can be removed right?

Comment (dannylin0711) [reply SpiceCompanion/Spice.entitlements:7]: Yes, also I forgot to remove the ".beta" in the Bundle Identifier.

**Comment by dannylin0711** [comment]
I push another update for fixing the bundle identifier, should automatically add to the pull request

**Comment by rodepanda** [comment]
Would you like to be added to the credits in the settings screen?
If so what name can I use?

**Comment by dannylin0711** [comment]
Use Lato please, this name should be fine.
And I forgot to remove the temporary code I'm testing moving to iOS 12 and below.
In the SpiceClient.swift, delete line 26-27, where the "connect" variable is declared.

**Pull request merged**